src/models/User.model.js.js

```diff
@@ -0,0 +1,25 @@
+const { DataTypes } = require('sequelize');
```

Comment:
The filename `User.model.js.js` seems to have a duplicate extension. Consider renaming the file to `User.model.js` for clarity and consistency.